src/parser/monk/mistweaver/modules/features/MistweaverHealingEfficiencyTracker.tsx

```diff
@@ -79,7 +79,7 @@ class MistweaverHealingEfficiencyTracker extends HealingEfficiencyTracker {
 
   getSoothingMistDetails(spellInfo: SpellInfoDetails) {
     // the default tracker gets the healing of the soothing mists, but only the mana for the first cast. Every tick costs mana.
-    spellInfo.manaSpent = this.soothingMist.soomTicks * SPELLS.SOOTHING_MIST.manaCost;
+    spellInfo.manaSpent = this.soothingMist.soomTicks * (SPELLS.SOOTHING_MIST.manaCost ? SPELLS.SOOTHING_MIST.manaCost : 1);
```

Comment:
This would break the accuracy of healer efficiency Tracker if manaCost is ever null. I'd rather have it error out for 30 minutes and fix it manually